_sass/_components/blog-index.scss

```diff
@@ -0,0 +1,111 @@
+// Blog index
```

Comment:
Could some of these break down to smaller components? Like RSS feed for example?